app/src/main/java/net/gsantner/markor/format/markdown/MarkdownTextModuleActions.java

```diff
@@ -67,6 +81,16 @@ public void appendTextModuleActionsToBar(ViewGroup barLayout) {
     //
     //
 
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
```

Comment:
Move this to base class (textmodulecations) and remove cast to MarkdownTextModuleActions